doc/source/ray-overview/getting-started.md

```diff
@@ -16,7 +16,7 @@ Use individual libraries for ML workloads. Click on the dropdowns for your workl
 `````{dropdown} <img src="images/ray_svg_logo.svg" alt="ray" width="50px"> Data: Scalable Datasets for ML
 :animate: fade-in-slide-down
 
-Scale offline inference and training ingest with [Ray Data](data_key_concepts) --
+Scale offline inference and training ingest with [Ray Data Quickstart](data_quickstart) --
```

Comment:
i think the actual title text should remain the same, but update the reference:
```suggestion
Scale offline inference and training ingest with [Ray Data](data_quickstart) --
```

Reply:
Updated.